style.css

```diff
@@ -53,8 +53,9 @@ body {
 
 .home-body {
   background-image: url(Assets/files/header\ shapes.png);
+  background-size: 100%;
   background-repeat: no-repeat;
-  min-width: 60%;
+  overflow: hidden;
   height: 604px;
 }
```

Comment:
|Project cards @ 768px| Project Cards @ >1440px| Others|
|:--------------:|:-----------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/253754264-07e3bf2c-bd3a-4f35-99c6-fdcca802c8f2.png)|![image](https://user-images.githubusercontent.com/99715640/253754379-ba8b012f-b649-4d8c-a049-06d857ada6d8.png)|
|About Design| Contact form Design|
|![image](https://user-images.githubusercontent.com/99715640/253754676-5630223e-73e1-41a0-b9bf-b83d5f58e139.png)|![image](https://user-images.githubusercontent.com/99715640/253755056-50caa308-a65f-4748-8dc4-c738578c8321.png)|
|Figma Design - Projects| About design - Figma| Contact form Design - Figma|
|![image](https://user-images.githubusercontent.com/99715640/253755222-06e6ee54-6402-4e75-90f9-966986cfbb05.png)|![image](https://user-images.githubusercontent.com/99715640/253755262-1006b0e3-8fad-417f-8f32-674fd8ca106a.png)|![image](https://user-images.githubusercontent.com/99715640/253755274-4be61d88-f596-41fe-9db6-2ad38f92cdb3.png)|

- As I mentioned earlier, you have made impressive progress on the mobile and everything is okay now. :+1: 
- [ ] Now, we just have to make a few adjustments to the desktop view in order to ensure a nice and responsive app. Please bear in mind that the ability to translate a medium or high-fidelity wireframe design into code with great precision according to details is a nice catch for recruiters or potential employers. This is why I think we should give it one more trial. :clap: You gat this!
- [ ] Kindly check and click on the images above for a broader view of the differences so that you can make appropriate changes to your code. :wrench: